docs/docs/dev/policies/traffic-metrics.md

```diff
@@ -297,9 +297,14 @@ This feature requires transparent proxy, so it's currently not available for Uni
 ::::
 
 ## Expose metrics from applications
-
-In addition to exposing metrics from the data plane proxies, you might want to expose metrics from applications running next to the proxies.
-
+ 
+In addition to exposing metrics from the data plane proxies, you might want to expose metrics from applications running next to the proxies. You can do this in 2 different ways:
+ 
+* directly from the applications,
+* through Kuma sidecar.
+ 
+### Expose metrics directly from the applications
```

Comment:
Do we really want to explain this? That has little to do with kuma.

Reply:
It's more about configuration, we don't really explain it too deeply. It's the same thing that was explained previously.

Reply:
I think it's just contributing to information overload. Let's remove this first option completely.